extensions/ql-vscode/package.json

```diff
@@ -542,6 +543,23 @@
     "@types/sinon-chai": "~3.2.3",
     "proxyquire": "~2.1.3",
     "@types/proxyquire": "~1.3.28",
-    "eslint-plugin-react": "~7.19.0"
+    "eslint-plugin-react": "~7.19.0",
+    "husky": "~4.2.5",
+    "lint-staged": "~10.2.2",
+    "prettier": "~2.0.5"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run lint-staged"
```

Comment:
this seems fine for now, but I am leaving a note here that husky appears to also have `pre-push` in case this turns out to feel to slow.

Reply:
Yep..I'm open to that if that's the direction you want to go.